### PR TITLE
[develop]: Add sample verification case w/instructions

### LINKED ISSUE #499: Sample Verification Cases
## Description
EPIC would like to provide sample verification cases that users can run through to (1) learn to use the basic verification features and (2) compare any forecast improvements/developments with a baseline SRW forecast to evaluate the extent to which the contribution improves forecasting capabilities. 
The current documentation does not contain any such sample cases or instructions. 

## Solution
Add a chapter to the documentation with an initial sample forecast case/instructions. Additional cases can be added with subsequent releases.

### PR DESCRIPTION
**Pull request by gspetro-NOAA**
## DESCRIPTION OF CHANGES: 
This PR includes a new chapter in the documentation for VX sample cases. Users can run through the case(s) to (1) learn to use the basic verification features and (2) compare any forecast improvements/developments with a baseline SRW forecast to evaluate the extent to which the contribution improves forecasting capabilities. Only one sample case is included in this PR, but others will be added subsequently. 

### Type of change
- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

## TESTS CONDUCTED: 
None required. Built the docs for this branch of my fork on RTD: https://srw-ug.readthedocs.io/en/text-vxcases/ .

- [x] hera.intel
- [ ] orion.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [x] gaea.intel
- [x] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)

## DEPENDENCIES:
N/A

## DOCUMENTATION:
All documentation.

## ISSUE: 
Issue #499 

## CHECKLIST

- [X] My code follows the style guidelines in the Contributor's Guide
- [X] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas N/A
- [X] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published N/A

## CONTRIBUTORS (optional): 
@natalie-perlin @EdwardSnyder-NOAA

### CODE REVIEW

File: docs/UsersGuide/source/VXCases.rst [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+-----------------------------
+
+First, navigate to the ``ufs-srweather-app/ush`` directory. Then, load the regional workflow environment:
+
+.. code-block:: console
+   
+   source <path/to/etc/lmod-setup.sh>
+   module use </path/to/ufs-srweather-app/modulefiles>
+   module load wflow_<platform>
+
```
Comment (MichaelLueken) [inline docs/UsersGuide/source/VXCases.rst:93]: Should either:

`conda activate regional_workflow`

or

`Please copy the command shown after loading the wflow_<platform> modulefile to load the regional_workflow conda environment.`

also be added to the end of this list of commands to load the regional workflow conda environment (at least on Level 1 platforms)?

**Review by MichaelLueken — COMMENTED** [review]
@gspetro-NOAA Excellent work!  I have one minor comment regarding the section under `Load the Regional Workflow`, but otherwise, everything looks good.

File: docs/UsersGuide/source/VXCases.rst (hunk 1 above)
Comment (gspetro-NOAA) [reply docs/UsersGuide/source/VXCases.rst:93]: Good catch! I updated accordingly. :)

**Review by MichaelLueken — APPROVED** [review]
@gspetro-NOAA Thanks!  Now approving this PR.

File: docs/UsersGuide/source/VXCases.rst [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+============================
+Verification Sample Cases
+============================
+
+Introduction
+===============
+
+The goal of these sample cases is to provide the UFS community with datasets that they can modify and run to see if their changes can improve the forecast and/or reduce the model biases. Each case covers an interesting weather event. The case that was added with the v2.1.0 release was a severe weather event over Indianapolis on June 15-16, 2019. In the future, additional sample cases will be provided. 
+
+Each sample case contains module output from a control run; this output includes ``postprd`` (post-processed) and ``metprd`` (MET verification-processed) directories. Under the ``postprd`` directory, users will find the :term:`UPP` output of the model run along with plots for several forecast variables. These can be used for a visual/qualitative comparison of forecasts. The ``metprd`` directory contains METplus verification statistics files, which can be used for a quantitative comparison of forecast outputs.
```
Comment (EdwardSnyder-NOAA) [inline docs/UsersGuide/source/VXCases.rst]: From the first line: 'Each sample case contains module'
'module' should be replaced with 'model'.

File: docs/UsersGuide/source/VXCases.rst [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+Comparing the plots is relatively straightforward since they are in ``.png`` format, and most computers can render them in their default image viewer. :numref:`Table %s <AvailablePlots>` lists plots that are available every 6 hours of the forecast (where ``hhh`` is replaced by the three-digit forecast hour): 
+
+.. _AvailablePlots:
+
+.. table:: Sample Indianapolis Forecast Plots
+
+   +-----------------------------------------+-----------------------------------+
+   | Field                                   | File Name                         |
+   +=========================================+===================================+
+   | Sea level pressure                      | slp_conus_fhhh.png                |
```
Comment (EdwardSnyder-NOAA) [inline docs/UsersGuide/source/VXCases.rst]: The png filenames changed from 'conus' to 'regional'. Example: slp_regional_fhhh.png
We did this so the background of the map will match with the forecast regional domain.

File: docs/UsersGuide/source/VXCases.rst [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+   task_get_extrn_ics:
+      # Add EXTRN_MDL_SOURCE_BASEDIR_ICS variable to config.yaml
+      EXTRN_MDL_SOURCE_BASEDIR_ICS: </path/to/Indy-Severe-Weather/input_model_data/FV3GFS/grib2/2019061500>
+      USE_USER_STAGED_EXTRN_FILES: true
+   task_get_extrn_lbcs:
+      # Add EXTRN_MDL_SOURCE_BASEDIR_LBCS variable to config.yaml
+      EXTRN_MDL_SOURCE_BASEDIR_LBCS:  </path/to/Indy-Severe-Weather/input_model_data/FV3GFS/grib2/2019061500>
+      USE_USER_STAGED_EXTRN_FILES: true
+   task_run_fcst:
+      WTIME_RUN_FCST: 03:00:00
```
Comment (natalie-perlin) [inline docs/UsersGuide/source/VXCases.rst]: Suggesting changing runtime to 5 hours at least to let the forecast to complete. Test on Gaea terminated after ~41h, due to 3 hour job limit

Comment (gspetro-NOAA) [reply docs/UsersGuide/source/VXCases.rst]: Done!

**Review by natalie-perlin — APPROVED** [review]

**Comment by MichaelLueken** [comment]
@EdwardSnyder-NOAA Just wanted to check base and make sure that you are happy with the changes that @gspetro-NOAA has made in this PR.  Please let @gspetro-NOAA know if there are any other changes that you would like for her to make.  Thanks!

**Comment by EdwardSnyder-NOAA** [comment]
LGTM! @MichaelLueken @gspetro-NOAA

**Pull request merged**